package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "ptvs",
-  "private": true,
-  "devDependencies": {
-    "@pylance/pylance": "2024.11.3"
-  }
+    "name":  "ptvs",
```

Comment:
formatting seems off? 

Also shouldn't there be an associated package-lock.json change?

Reply:
We don't use package-lock.json in PTVS. This file gets updated automatically whenever you run PreBuild.ps1 and a new version of pylance is downloaded.